clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerProtocolTest.java

```diff
@@ -153,7 +153,7 @@ public void deserializeOldSubscriptionVersion() {
         Subscription subscription = new Subscription(Arrays.asList("foo", "bar"), null);
         ByteBuffer buffer = ConsumerProtocol.serializeSubscription(subscription, (short) 0);
         Subscription parsedSubscription = ConsumerProtocol.deserializeSubscription(buffer);
-        assertEquals(toSet(parsedSubscription.topics()), toSet(parsedSubscription.topics()));
+        assertEquals(Set.copyOf(parsedSubscription.topics()), Set.copyOf(parsedSubscription.topics()));
```

Comment:
This should be a wrong unit tests, expect code should be ` assertEquals (Set) copyOf (subscription. The switchable viewer ()), Set the copyOf (parsedSubscription. Switchable viewer ())); `

Reply:
yes, you are right. Could you please fix it in this PR?

Reply:
Of course, I have pushed the repair code, please have a look